tests/utils.py

```diff
@@ -2,10 +2,33 @@
 
 import pytest
 from fastapi._compat import PYDANTIC_V2
+from inline_snapshot import Snapshot
 
 needs_py39 = pytest.mark.skipif(sys.version_info < (3, 9), reason="requires python3.9+")
 needs_py310 = pytest.mark.skipif(
     sys.version_info < (3, 10), reason="requires python3.10+"
 )
 needs_pydanticv2 = pytest.mark.skipif(not PYDANTIC_V2, reason="requires Pydantic v2")
 needs_pydanticv1 = pytest.mark.skipif(PYDANTIC_V2, reason="requires Pydantic v1")
+
+
+def pydantic_snapshot(
+    *,
+    v2: Snapshot,
+    v1: Snapshot,  # Remove v1 argument when deprecating Pydantic v1
```

Comment:
```suggestion
    v1: Snapshot,  #TODO: remove v1 argument when deprecating Pydantic v1
```
(in same format as other TODO's to facilitate simple ctrl-F'ing 😉 )